test/graphql/Offer/queries/getMyOffers.test.ts

```diff
@@ -29,150 +33,171 @@ const GET_MY_OFFERS = gql`
+    expiredOfferForStudents.expire();
+    expiredOfferForBoth.expire();
+    await OfferRepository.save(expiredOfferForGraduates);
+    await OfferRepository.save(expiredOfferForStudents);
+    await OfferRepository.save(expiredOfferForBoth);
+    allOffers = [
+      offers[ApplicantType.student][ApprovalStatus.pending],
+      offers[ApplicantType.student][ApprovalStatus.approved],
+      offers[ApplicantType.student][ApprovalStatus.rejected],
+      offers[ApplicantType.graduate][ApprovalStatus.pending],
+      offers[ApplicantType.graduate][ApprovalStatus.approved],
+      offers[ApplicantType.graduate][ApprovalStatus.rejected],
+      offers[ApplicantType.both][ApprovalStatus.pending],
+      offers[ApplicantType.both][ApprovalStatus.approved],
+      offers[ApplicantType.both][ApprovalStatus.rejected],
+      expiredOfferForGraduates,
+      expiredOfferForStudents,
+      expiredOfferForBoth
+    ];
+    allOffers = allOffers.sort(offer => -offer.updatedAt);
+    allOfferUuids = allOffers.map(({ uuid }) => uuid);
+  });
 
-    beforeAll(async () => {
-      const companyClient = await TestClientGenerator.company({ status: ApprovalStatus.approved });
-      apolloClient = companyClient.apolloClient;
-      company = companyClient.company;
-    });
+  const getMyOffers = (variables: IGetMyOffers, apolloClient?: TestClient) =>
+    (apolloClient || companyApolloClient).query({ query: GET_MY_OFFERS, variables });
 
-    it("returns only the offers that the company made, by updatedAt desc", async () => {
-      await createOffers(company.uuid);
-      const { data, errors } = await apolloClient.query({
-        query: GET_MY_OFFERS
-      });
-      expect(errors).toBeUndefined();
-      expect(data!.getMyOffers.shouldFetchMore).toEqual(false);
-      expect(data!.getMyOffers.results).toHaveLength(2);
-      expect(data!.getMyOffers.results).toMatchObject([
-        { uuid: offer2.uuid },
-        { uuid: offer1.uuid }
-      ]);
-    });
+  it("returns all the offers from the current company", async () => {
+    const { data, errors } = await getMyOffers({ hideRejectedAndExpiredOffers: false });
+    expect(errors).toBeUndefined();
+    const { shouldFetchMore, results } = data!.getMyOffers;
+    const resultUuids = results.map(({ uuid }) => uuid);
+    expect(shouldFetchMore).toEqual(false);
+    expect(resultUuids).toEqual(expect.arrayContaining(allOfferUuids));
+  });
 
-    describe("pagination", () => {
-      let newOffersByDescUpdatedAt: Offer[] = [];
-
-      beforeAll(async () => {
-        for (const _ of range(15)) {
-          newOffersByDescUpdatedAt.push(
-            await OfferGenerator.instance.withObligatoryData({
-              companyUuid: company.uuid
-            })
-          );
-        }
-        newOffersByDescUpdatedAt = newOffersByDescUpdatedAt.sort(offer => -offer.updatedAt);
-      });
+  it("returns the offers that are approved or pending for graduates or students", async () => {
```

Comment:
aca tmb, no se esta mencionando "expired" o "not expired" en las descripciones de los tests

Reply:
porque no hay que me mencionarlos

Reply:
no retorna ofertas expiradas, por eso no se mencoona